publishers/sample/sample_demisto.py

```diff
@@ -0,0 +1,55 @@
+"""
+Batch of example publishers usable with Demisto.
+"""
+from streamalert.shared.publisher import Register
+
+
+@Register
+def demisto_classification(alert, publication):
+    """
+    This publisher appropriately sets the demisto incident type and playbook.
+
+    It first looks into the alert's context for the "demisto" key, where individual rules can
+    explcitly specify the desired classification traits of the output alert.
+    """
+
+    # If a rule explicitly states Demisto information with the alert context, obey that
+    # The convention to follow is any key in this dict (example, "incident_type") is mapped
+    # directly onto the Demisto output magic keys (example, @demisto.incident_type)
+    if 'demisto' in alert.context:
```

Comment:
We can use a const for `demisto` string which is referenced in multiple places. It is not only make code clean but also reduce the work if `demisto` renames itself in the near future.